sympy/physics/mechanics/tests/test_rigidbody.py

```diff
@@ -18,12 +18,12 @@ def test_rigidbody_default():
     assert b.__str__() == 'B'
     assert b.__repr__() == (
         "RigidBody('B', masscenter=B_masscenter, frame=B_frame, mass=B_mass), "
-        "inertia=(B_ixx*(B_frame.x|B_frame.x) + "
+        "inertia=Inertia(dyadic=B_ixx*(B_frame.x|B_frame.x) + "
         "B_ixy*(B_frame.x|B_frame.y) + B_izx*(B_frame.x|B_frame.z) + "
         "B_ixy*(B_frame.y|B_frame.x) + B_iyy*(B_frame.y|B_frame.y) + "
         "B_iyz*(B_frame.y|B_frame.z) + B_izx*(B_frame.z|B_frame.x) + "
         "B_iyz*(B_frame.z|B_frame.y) + B_izz*(B_frame.z|B_frame.z), "
-        "B_masscenter)))")
+        "point=B_masscenter)))")
```

Comment:
This is a breaking change. Can we leave it as it was?

Reply:
Would not say that improving the `repr` is an breaking change. Especially since this one has already been changed less than 2 months ago or so.

Reply:
Ok, I didn't no it has been changed. Input and output are API, so changing output is breaking. What was the change 2 months ago?

Reply:
https://github.com/sympy/sympy/commit/1ec2c5ae33fcef0944d293d3a40d589126d79d8a#diff-ca6cdf21e7f3c3c4901a45bfe5eee7c13c36d59bde549fbf3075912689fa0cf1 Prior to this commit the `__repr__` returned the same as `__str__`. The change was to have the `__repr__` instead produce a more complete output of the instance and its attributes.

Reply:
Ok, chances it will effect something are probably quite small. I won't hold it up.